data/human/human missions.txt

```diff
@@ -7455,3 +7455,1836 @@ mission "Adelita 5"
+			`	You spend the day in the hospital room chatting with with Timothy, before finally having to leave.`
+			`	"Thank you for coming here. I wish I had something to give you, but I own nothing anymore, even though I owe you everything. Take care out there, and I'll do my best to keep out of trouble in here."`
+			label goodbyes
+			choice
+				`	(Offer to transfer the Hero of Rand award and stipend to Timothy.)`
+					goto offer
+					to display
+						not "offers"
+				`	(Shake his hand and say goodbye.)`
+					goto shake
+				`	(Give him a hug and say goodbye.)`
+
+			action
+				clear "offers"
+			`	You lean over and give him partial hug in his sick bed. He wraps his one good arm around you and squeezes. When he finally breaks free you see a tear.`
+			`	You leave him bedridden in the hospital, and to your surprise you find your guide waiting for you outside the room. "His is a tough case. Perhaps one day he will recover enough to leave this place." The monk pauses, thoughtfully. "Or perhaps not."`
+			`	He leads you back to the train station, and from there you take brief journey back to New Tibet's spaceport.`
+				decline
+
+			label shake
+			action
+				clear "offers"
+			`	You lean in to shake his good hand. He swallows and clears his throat, trying to keep himself from crying.`
+			`	You leave him bedridden in the hospital, and to your surprise find your guide waiting for you outside the room. "His is a tough case. Perhaps one day he will recover enough to leave this place." The monk pauses, thoughtfully. "Or perhaps not."`
+			`	He leads you back to the train station, and from there you take brief journey back to New Tibet's spaceport.`
+				decline
+		
+			label offer
+			action
+				set "offers"
+			`	Timothy's eyes go wide. "No, you couldn't. It's yours. You've given me too much already."`
+			choice
+				`	(Tell him about what Wynnafrith said, and how he is the real Hero of Rand.)`
+					goto offer2
+				`	(Tell him you don't need the money, but he does.)`
+					goto offer2
+				`	(Agree with him and rescind your offer.)`
+					goto goodbyes
+			
+			label offer2
+			action
+				clear "salary: Hero of Rand"
+				clear "offers"
+			`	As you speak Timothy shakes his head slowly, tears streaming down his face. Finally, as you finish, he reaches out to you with his one good arm and pulls you in for a hug.`
+			`	He tries to speak, but his voice fails him. Instead he just shakes his teary-eyed head and bows it before you in deep respect.`
+			`	"Always." He manages to whisper through phlegm and weakened lungs as you turn to leave, "always you."`
+			`	You leave him bedridden in the hospital, and to your surprise find your guide waiting for you outside the room.`
+			`	"His is a tough case. Perhaps one day he will recover enough to leave this place." The monk pauses, thoughtfully. "Or perhaps not."`
+			`	He leads you back to the train station, and from there you take the brief journey back to New Tibet's spaceport.`
+				decline
```

Comment:
```suggestion
```
Otherwise
![image](https://github.com/user-attachments/assets/517f8dc0-1539-4542-aff4-97d42a98bfa1)

Reply:
That's odd, but I suppose that `decline` node is unnecessary.